product-service/serverless.yaml

```diff
@@ -6,6 +6,13 @@ provider:
   stage: dev
   region: eu-central-1
 
+environment:
+  PG_HOST: lesson4.czoz6duffqoz.eu-central-1.rds.amazonaws.com
+  PG_PORT: 5432
+  PG_DATABASE: lesson4
+  PG_USERNAME: postgres
+  PG_PASSWORD:
+
```

Comment:
This data should not be committed into the repository. Anyway, It should be used in the codebase via `proccess.env`, whereas hardcoded values are used.